plugins/PostProcessingPlugin/scripts/MultiExtColorMix.py

```diff
@@ -0,0 +1,623 @@
+            enable_invis = False
+        self._instance.setProperty("invis_setting", "value", enable_invis)
+    # Post a message if only a single instance of this post is running-------------------------------------------
+        multi_present = 0
+        pp_name_list = Application.getInstance().getGlobalContainerStack().getMetaDataEntry("post_processing_scripts")
+        for pp_name in pp_name_list.split("\n"):
+            if "MultiExtColorMix" in pp_name:
+                multi_present += 1
+        if multi_present == 1:
+            Message(text = "Multi-Extruder Color Mixer:" + "\n" + "You must have a Blending Hot End.  Extruders Share Heater' and 'Extruders Share Nozzle' must be enabled in the 'Printer Settings' in Cura.  Open 'MultiExtColorMix.py' in a text editor and review the opening lines for more instructions." + "\n").show()
+            
+    def getSettingDataString(self):
+        return """{
+            "name":"Multi-Extruder Color Mixer",
+            "key":"MultiExtColorMix",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "mix_style":
+                {
+                    "label": "Mix Style",
+                    "description": "'Constant' is NO CHANGE.  'Gradient' is VARIABLE from Start Mix percentage to End Mix percentage across the spread of layers. If you have 3 extruders you can include two or all three.  The 'Start Mix' numbers must add up to 100.  The 'End Mix' numbers must add up to 100.",
+                    "type": "enum",
+                    "options": {"constant":"Constant","gradient":"Gradient"},
+                    "default_value": "constant"
+                },
+                "start_layer":
+                {
+                    "label": "Start Layer",
+                    "description": "Layer to start mixing.  Use the Cura preview number.",
+                    "type": "int",
+                    "default_value": 1,
+                    "minimum_value": 1
+                },
+                "end_layer":
+                {
+                    "label": "End Layer",
+                    "description": "Layer to end the mixing.  Use -1 for the top layer or the Cura preview layer you wish to end at.",
+                    "type": "int",
+                    "default_value": -1,
+                    "minimum_value": -1
+                },
+                "resume_ext":
+                {
+                    "label": "    Resume Extruder",
+                    "description": "The extruder to use after mixing ends.  The mixing extruder is 'Number of Extruders' + 1 (regardless of whether they are enabled or not, if the printer has 3 extruders then the mixer is Extruder #4).",
+                    "type": "int",
+                    "default_value": 1,
+                    "minimum_value": "1",
+                    "maximum_value": "4",
+                    "enabled": "end_layer != -1"
+                },
+                "T0_include":
```

Comment:
we never capitalize setting keys
```suggestion
                "t0_include":
```